src/FixedDataTableBufferedRows.js

```diff
@@ -155,9 +155,9 @@ class FixedDataTableBufferedRows extends React.Component {
     const style = {};
     FixedDataTableTranslateDOMPosition(style, 0, containerOffsetTop, false);
 
-    // NOTE (pradeep): Sort the rows by row index so that they appear with the right order in the DOM (see #221)
-    const sortedRows = sortBy(this._staticRowArray, (row) =>
-      get(row, 'props.ariaRowIndex', Infinity)
+    // // NOTE (pradeep): Sort the rows by row index so that they appear with the right order in the DOM (see #221)
+    const sortedRows = this._staticRowArray.sort(
+      (row) => row?.props?.ariaRowIndex || Infinity
```

Comment:
lodash's `sortBy` and `get` is a bit slower compared to native sort and optional chaining operations.